pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/mergerollup/segmentgroupmananger/MergeRollupTaskSegmentGroupManagerProvider.java

```diff
@@ -27,20 +27,27 @@
  * Provider class for {@link MergeRollupTaskSegmentGroupManager}
  */
 public abstract class MergeRollupTaskSegmentGroupManagerProvider {
+
+  private static String _defaultMergeRollupTaskSegmentGroupManagerClassName =
+      "DefaultMergeRollupTaskSegmentGroupManager";
```

Comment:
Doesn't the plugin require fully qualified class name?

Reply:
Good catch! updated.